packages/runtime/container-runtime/src/gc/gcUnreferencedStateTracker.ts

```diff
@@ -25,6 +25,54 @@ class TimerWithNoDefaultTimeout extends Timer {
 	}
 }
 
+/** The collection of UnreferencedStateTrackers for all unreferenced nodes */
+export class UnreferencedNodeTrackers extends Map<string, UnreferencedStateTracker> {
+	constructor(private readonly configs: IGarbageCollectorConfigs) {
```

Comment:
nit: Pass only the properties needed from configs here. That will help writing unit tests easier.

Reply:
N/A